src/txDigester/txDigestFunctions.ts

```diff
@@ -0,0 +1,94 @@
+import { config } from '../Config'
+import * as processedTxs from '../dbstore/processedTxs'
+import * as txDigest from './txDigests'
+import * as Crypto from '../Crypto'
+
+let lastProcessedTxDigest: txDigest.TransactionDigest = null
+
+export interface txDigestObj {
+  prevHash: string
+  txIdsHash: string
+}
+
+export const getTxIds = async (startCycle: number, endCycle: number): Promise<string[]> => {
+  const sortedTxIds = await processedTxs.querySortedTxsBetweenCycleRange(startCycle, endCycle)
+  return sortedTxIds
+}
+
+export const getHash = async (cycle: number): Promise<string> => {
+  const txDigestHash = await txDigest.queryByEndCycle(cycle)
+  return txDigestHash ? txDigestHash.hash : '0x0'
```

Comment:
we should have some checks here. Like return '0x0' only if cycle = 0. If not, then throw error.

Reply:
Done